account-kit/rn-signer/ios/implementation/KeychainHelper.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import Security
+import CryptoKit
+
+let account_kit_ephemeral_private_key = "ephemeralPrivateKey"
+
+// Keep save_ephemeral_private_key_query mutable
+var save_ephemeral_private_key_query: [String: Any] = [
+    kSecClass as String: kSecClassGenericPassword,
+    kSecAttrAccount as String: account_kit_ephemeral_private_key,
+    kSecAttrAccessible as String: kSecAttrAccessibleAfterFirstUnlock
+]
+
+let get_ephemeral_private_key_query: [String: Any] = [
+    kSecClass as String: kSecClassGenericPassword,
+    kSecAttrAccount as String: account_kit_ephemeral_private_key,
+    kSecReturnData as String: true,
+    kSecMatchLimit as String: kSecMatchLimitOne
+]
+
+let delete_ephemeral_private_key_query: [String: Any] = [
+    kSecClass as String: kSecClassGenericPassword,
+    kSecAttrAccount as String: account_kit_ephemeral_private_key
+]
+
+
+class KeychainHelper {
```

Comment:
Also tiny nit on the naming KeyChainHelper might be too generic and confuse with the library name. Let's call it  `PrivateKeyChainUtilities`